pnt-lut.c

```diff
@@ -62,21 +68,36 @@ int main(int ac, char *av[]) {
     mkfifo(pnt_fifo, 0666);
     const int fd = open(pnt_fifo, O_RDONLY);
     const int fd_out = open(output_fifo, O_WRONLY);
-    uint8_t pnt[2];
+    int pnt[2];
     for (;;) {
       const int rv = read(fd, pnt, sizeof(pnt));
```

Comment:
it may be worth it to use `better_read` here, now that we're reading 8 bytes. Probably won't ever chunk that, but it wouldn't exactly hurt either

Reply:
agreed